### ISSUE TITLE
Unable to approve a site page with Approval flow enabled

### ISSUE THREAD

**Issue by vspinelli** [SharePoint/sp-dev-docs#9641]
### Target SharePoint environment

SharePoint Online

### What SharePoint development model, framework, SDK or API is this about?

💥 SharePoint Framework

### Developer environment

None

### What browser(s) / client(s) have you tested

- [ ] 💥 Internet Explorer
- [X] 💥 Microsoft Edge
- [X] 💥 Google Chrome
- [ ] 💥 FireFox
- [ ] 💥 Safari
- [ ] mobile (iOS/iPadOS)
- [ ] mobile (Android)
- [ ] not applicable
- [ ] other (enter in the "Additional environment details" area below)

### Describe the bug / error

In all our customers sites (diifferent tenant), where we have enabled the [Approval flow for modern pages](https://support.microsoft.com/en-us/office/approval-flow-for-modern-pages-a8b2e689-d4a1-4639-8028-333c0ece30d9),
since few days, when we try to approve a sitepage, after clicked on "Review approvals" button, the panel loads infinitely, not allowing us to approve the page.

![image](https://github.com/SharePoint/sp-dev-docs/assets/16682138/296a6977-429a-4778-91f9-d1b24ef18ee5)
 

We get an error message in the browser console:

**Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self' https://content.powerapps.com/ https://static.powerapps.com/ https://*.cdn.office.net https://wcpstatic.microsoft.com/ https://*.msftauth.net https://*.res.office365.com https://*.monitor.azure.com https://*.flow.microsoft.com 'sha256-...' 'sha256-...' 'sha256-...'"**

The issue is not present if the page is approved directly in a email

### Steps to reproduce

1. Enable the approval flow on SitePages
2. Create a page and send a request to approve it
3. Approval editor try to approve the request directly on a SharePoint page

### Expected behavior

The approval panel shows the actions to approve or reject the request

**Comment by mkm17** [comment]
Hi @vspinelli, could you also check the Network tab of your browser's console and review the requests there? I've checked it on my end, and the following requests are triggered.

![image](https://github.com/SharePoint/sp-dev-docs/assets/17220108/0b236ba5-23a0-4e6b-b01f-967ad9f7ea89)

**Comment by vspinelli** [comment]
Hi @mkm17, have you enabled the approval flow on SItePages library? From the screenshot you have provided, it appears that it has not been enabled.

This is the error displayed in the Network tab
![image](https://github.com/SharePoint/sp-dev-docs/assets/16682138/d8d137c2-397b-4b2a-8b45-7f4f4ff32a3c)

And this is the details of the request:
`{"csp-report":{"document-uri":"https://make.powerautomate.com/it-IT/widgets/manage/environments/Default-aec017bb-e5f4-43b6-900c-df361533110e/approvalcenter/received?widgetId=e6b7e4c8-e3a9-4b2c-aafa-1c91-9fd64aed&sdkVersion=2.1&widgetType=approvalCenter&utm_medium=widget&utm_source=SPO&environment=Default-aec017bb-e5f4-43b6-900c-df361533110e&enableOnBehalfOfTokens=true&enableRegionalPortal=true&enableWidgetV2=true&oAuthHandler=oAuth&backgroundColor=white&hideTabs=false&approvalsFilter=properties%2FitemLinkEncoded%20eq%20%27aHR0cHM6Ly9hY2lucXVlLnNoYXJlcG9pbnQuY29tL3NpdGVzL3Rlc3R5b3VuaXR5LmFjaW5xdWUuaXQvU2l0ZVBhZ2VzL0hvbWUuYXNweD9ldGFnPSUyMiU3QkU1NjQ2QkQyLTNERjMtNEU5OS05QzI5LTgyNkUzRjBCRTM3NSU3RCUyQzQ1MyUyMg%3D%3D%27&autoNavigateToDetails=true&hideInfoPaneCloseButton=true&hideFlowCreation=true&showSimpleEmptyPage=true&hideLink=true&parameters.sharepoint.site=%5Bobject%20Object%5D&parameters.sharepoint.list=abfacce6-0d1d-4e91-921d-84ab6f83be69&useServerSideProvisioning=true&pathAlias=receivedApprovals&allowOptionalEvents=true","referrer":"https://acinque.sharepoint.com/","violated-directive":"script-src","effective-directive":"script-src","original-policy":"base-uri 'self'; media-src 'none'; object-src 'none'; manifest-src 'self'; script-src 'self' https://content.powerapps.com https://static.powerapps.com https://*.cdn.office.net https://wcpstatic.microsoft.com https://*.msftauth.net https://*.res.office365.com https://*.monitor.azure.com https://*.flow.microsoft.com 'sha256-XXXXXXXXX' 'sha256-XXXXXXXXXXXX' 'sha256-XXXXXXX'; style-src 'unsafe-inline'; report-uri https://csp.microsoft.com/report/PowerAutomate-MakerPortal;","disposition":"report","blocked-uri":"eval","line-number":2,"column-number":15619,"source-file":"https://content.powerapps.com/resource/makerx/static/js/module/245.5f353f70.chunk.js","status-code":200,"script-sample":""}}`

In any case, today we notice that after some time, after about 2 minutes of waiting, the approval popup appears

**Comment by mkm17** [comment]
Hi @vspinelli, I apologize, you were correct. I have just checked the standard approval process. 

I compared my requests to yours, and I see the same error in the console, but the form loads after some time. It definitely takes less than two minutes, around 7 to 15 seconds. In my case, I don't have any errors in the network tab.

![image](https://github.com/SharePoint/sp-dev-docs/assets/17220108/1173437c-8312-418b-8911-1da759042da0)

![image](https://github.com/SharePoint/sp-dev-docs/assets/17220108/8e4e69ee-1766-4c62-8178-44dedf4fe136)

As it seems there might be a problem with the Power Automate endpoint `https://make.powerautomate.com/it-IT/widgets/manage/environments/`, it could be beneficial to seek support from the Admin center or the Power Automate center to check for any performance issues with this service.
